examples/step-79/doc/intro.dox

```diff
@@ -0,0 +1,597 @@
+in which case the stress-strain tensor can be expressed in terms of the Lam\'{e}
+ parameters $\lambda,\mu$ such that
+@f[
+  \sigma = \rho^p (\lambda tr(\epsilon) I + 2 \mu \epsilon)
+@f]
+@f[
+  \sigma_{i,j} = \rho^p (\lambda  \epsilon_{k,k} \delta_{i,j}
+  + 2 \mu \epsilon_{i,j})
+@f]
+see step-8 for how this transformation works.
+
+Integrating the objective function by parts, along with using the elasticity
+equation constraint, gives a simplification of the objective problem as follows:
+
+@f[
+  \int_\Omega \sigma(\rho) : (\nabla \textbf{u} + (\nabla \textbf{u})^T) +
+  \int_\Omega (\nabla \cdot \sigma(\rho)) \cdot \textbf{u} =
+  \int_{\partial \Omega} \textbf{t} \cdot \textbf{u}
+@f]
+
+@f[
+  \int_\Omega \sigma(\rho) : (\nabla \textbf{u} + (\nabla u)^T) =
+  \int_\Omega \textbf{F}\cdot \textbf{u} +
+  \int_{\partial \Omega} \textbf{t} \cdot \textbf{u}
+@f]
+
+Because we are assuming no internal forces, this simplifies all the way to
+
+@f[
+  \int_\Omega \sigma(\rho) : (\nabla \textbf{u} + (\nabla \textbf{u})^T)
+  = \int_{\partial \Omega} \textbf{t} \cdot \textbf{u}
+@f]
+
+<h3>Making the solution mesh-independent</h3>
+
+Typically, the solutions to topology optimization problems are not
+mesh-independent, and as such the problem is ill-posed. This is because as the
+mesh is refined further, fractal structures are often formed. As the mesh gains
+resolution, the optimal solution typically gains smaller and smaller structures.
+There are a few competing work-arounds to this issue, but the most popular for
+first order optimization is the sensitivity filter, while second order
+optimization methods tend to prefer use of a density filter.
+
+As the filters affect the gradient and Hessian of the compliance (i.e., the
+objective function), the choice of filter has an effect on the solution of the
+problem. The density filter as part of a second order method works by
+introducing an unfiltered density, which we refer to as $\sigma$, and then
+requiring that the density be a convolution of the unfiltered density:
+@f[
+  \rho = H(\sigma).
+@f]
+Here, $H$ is an operator so that $\rho(\mathbf x)$ is some kind of average of
+the values of $\sigma$ in the area around $\mathbf x$ -- i.e., it is a smoothed
+version of $\sigma$.
+
+This prevents checkerboarding; the radius of the filter allows the user to
+define an effective minimal beam width for the optimal structures we seek to
+find.
+
+<h3>Complete Problem Formulation</h3>
+
+The minimization problem is now
+@f[
+  \min_{\rho,\sigma,\mathbf u} \int_{\partial\Omega} \textbf{u} \cdot \textbf{t}
+@f]
+@f[
+  \text{s.t.   } \rho = H(\sigma)
+@f]
+@f[
+  \int_\Omega \rho^p \left(\frac{\mu}{2}\left(\epsilon(\textbf{v}):
+  \epsilon(\textbf{u})) \right) + \lambda \left( \nabla \cdot \textbf{u} \nabla
+  \cdot \textbf{v} \right)  \right) = \int_{\partial \Omega} \textbf{v} \cdot
+  \textbf{t}
+@f]
+@f[
+  \int_\Omega \rho = V
+@f]
+@f[
+  0\leq \sigma \leq 1
```

Comment:
Which `\sigma` is this?

Reply:
The not bold one. :) The cauchy stress only appears in the introductory part of the intro (making it very clear to me which I was using before I shoved it all together and realized I used the same thing twice)